fe/fe-core/src/main/java/org/apache/doris/catalog/ResourceMgr.java

```diff
@@ -163,6 +163,11 @@ public boolean containsResource(String name) {
     }
 
     public Resource getResource(String name) {
+        // nameToResource == null iff this is in replay thread
```

Comment:
iff ==>if

Reply:
This is not a typo, `iff` means `if and only if`